analytics_data_api/v0/views/courses.py

```diff
@@ -763,3 +766,37 @@ class VideosListView(BaseCourseView):
     def apply_date_filtering(self, queryset):
         # no date filtering for videos -- just return the queryset
         return queryset
+
+
+class ReportDownloadView(APIView):
+    """
+    Get information needed to download a CSV report
+
+    **Example request**
+
+        GET /api/v0/courses/{course_id}/reports/{report_name}/
+
+    **Response Values**
+
+        Returns a single object with data about the report, with the following data:
+
+            * course_id: The ID of the course
+            * report_name: The name of the report
+            * download_url: The Internet location from which the report can be downloaded
+
+        The object may also return these items, if supported by the storage backend:
+
+            * last_modified: The date the report was last updated
+            * expiration_date: The date through which the link will be valid
+            * file_size: The size in bytes of the CSV download
+    """
+    enabled_reports = (
+        'problem_response'
+    )
```

Comment:
I know you mentioned this in your description, but I think we should at the very least move this tuple to a django setting. Then we could make that setting pull from `analytics_api.yml` in base.py.